--- a/scripts/northrend/icecrown_citadel/frozen_halls/pit_of_saron/boss_forgemaster_garfrost.cpp	2012-11-17 16:58:08.932286039 +0100
+++ b/scripts/northrend/icecrown_citadel/frozen_halls/pit_of_saron/boss_forgemaster_garfrost.cpp	2012-11-17 19:28:58.870343623 +0100
@@ -54,8 +54,8 @@
     SPELL_FORGE_BLADE                   = 68774,
     SPELL_FORGE_BLADE_H                 = 70334,
 
-    SAY_FREE_SLAVE_HORDE                = -1658013,
-    SAY_FREE_SLAVE_ALLY                 = -1658012,
+    SAY_FREE_SLAVE_HORDE                = -1658068,
+    SAY_FREE_SLAVE_ALLY                 = -1658021,
     SAY_TYRANNUS_OUTRO                  = -1658020,
 
     PHASE_NO_ENCHANTMENT                = 1,